doc/release/README.chplenv

```diff
@@ -361,12 +363,12 @@ Optional Settings
 
    If unset, CHPL_HWLOC defaults to "hwloc" if CHPL_TASKS is "qthreads",
    unless the target platform is knc.  In all other cases it defaults to
-   "none".  In the unlikely event the bundled hwloc distribution does
+   "none". In the unlikely event the bundled hwloc distribution does
    not build successfully, it should still be possible to use qthreads.
    Manually set CHPL_HWLOC to "none" and rebuild in this case (and
    please file a bug with the Chapel team.)  Building without hwloc
    should not have a large performance impact when CHPL_LOCALE_MODEL
-   is "flat" but will drastically hurt performance for "numa".
+   is "flat", but will drastically hurt performance for "numa".
```

Comment:
Reviewing open PRs going into the release:  I believe this comma is technically incorrect since the clause following it does not have its own subject but shares "Building without hwloc" with the first clause.